### DISCUSSION TITLE
General Chat

### DISCUSSION THREAD

**Discussion by reisemh3** [reisemh3/applications-csharp#13] [General]
## 👋 Welcome in General Chat!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Talk to everything you want.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dkm94** [comment]
Vous préférez créer des branches à nos noms ou une branche par feature/tâche ?
